src/TemporaryUploadedFile.php

```diff
@@ -142,20 +142,51 @@ public static function createFromLivewire($filePath)
 
     public static function canUnserialize($subject)
     {
-        if (! is_string($subject)) return false;
+        if (is_string($subject)) {
+            return Str::startsWith($subject, 'livewire-file:')
+                || Str::startsWith($subject, 'livewire-files:');
+        } elseif (is_array($subject)) {
+            return collect($subject)->filter(function ($key) {
```

Comment:
First, a ->filter() without any args should work fine for what you want.

Second, why do you even need a filter here at all? I don't think you do.

Reply:
@calebporzio  A filter is required to skip all null string, if not `Str::startsWith` will throw an exception.